runtime/jcl/common/acccont.c

```diff
@@ -25,6 +25,7 @@
 #include "j9.h"
 #include "j9port.h"
 
+#if JAVA_SPEC_VERSION < 24
```

Comment:
This file can be removed for JDK24+.

Reply:
How do I do that?

Reply:
It can be decorated w/ `if(JAVA_SPEC_VERSION LESS 24)`
https://github.com/eclipse-openj9/openj9/blob/7576b59bcd086c90f32fc0115995d15d965ce024/runtime/jcl/CMakeLists.txt#L109

Reply:
Thanks, I added that and removed the directives from acccont.c.